### ISSUE TITLE
bind source path does not exist

### ISSUE THREAD

**Issue by fangzongzhou** [kubeshop/tracetest#2134]
**Describe the bug**
when i follow the step from https://docs.tracetest.io/getting-started/installation/  

the result is ` bind source path does not exist`



**To Reproduce**
exec 
```
docker compose -f tracetest/docker-compose.yaml  up -d
```


**Screenshots**
output is 
```
➜  workspace docker compose -f tracetest/docker-compose.yaml  up -d
[+] Running 3/3
 ⠿ demo-worker Pulled                                                                                                                                                                                                                                                                                                                                                  4.1s
 ⠿ demo-api Pulled                                                                                                                                                                                                                                                                                                                                                     3.7s
 ⠿ demo-rpc Pulled                                                                                                                                                                                                                                                                                                                                                     3.7s
[+] Running 0/0
 ⠋ Container tracetest-tracetest-1  Creating                                                                                                                                                                                                                                                                                                                           0.1s
Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /host_mnt/Users/fangzongzhou/workspace/tracetest/tracetest-provision.yaml
```




**Additional context**
my device is mac M1 

docker desktop 

```
Version 4.17.0 (99724)

Engine: 20.10.23

Compose: v2.15.1

Credential Helper: v0.7.0

Kubernetes: v1.25.4
```

**Comment by danielbdias** [comment]
Thanks for reporting it @fangzongzhou . We are checking this bug and fixing it this week. As soon as release a version we will let you know!

**Comment by danielbdias** [comment]
hi, @fangzongzhou! We released version `v0.10.1` today with a bug fix for this problem. We you have time, can you update `tracetest` CLI through `brew upgrade tracetest` and try again?

**Comment by danielbdias** [comment]
hi folks! I'll close this issue now since it is fixed on `v0.10.1`.  Feel free to ping us in case of any doubt!
